indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,269 @@
+    {% if contrib.subcontributions %}
+        <div>
+            <b>{% trans %}Subcontributions{% endtrans %}</b>
+            <ul class="sub-contrib-timeline inherited-bg-color-trans inherited-border-color">
+                {% for subchild in contrib.subcontributions %}
+                    <li class="inherited-border-color">
+                        <div class="contrib-heading">
+                            {% if show_length_contribs and not contrib.session.is_poster %}
+                                <span class="timebox inherited-bg-color inherited-txt-color">
+                                    {{ (subchild.duration)|format_human_timedelta(narrow=True) }}
+                                </span>
+                            {% endif %}
+                            <h4>{{ subchild.title }}</h4>
+                        </div>
+                        <div class="contrib-timeline-content">
+                            {{ render_nested_contrib(subchild, show_abstract, dont_show_poster_abstract, show_title, show_affiliation, show_length_contribs) }}
+                        </div>
+                    </li>
+                {% endfor%}
+            </ul>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_block_date(entry, day, print_date_close_to_sessions) %}
+    {% if print_date_close_to_sessions %}
+        <p class="session-block-date">
+            <b>{{ day|format_skeleton('EEEEdMMMM') }}</b>
+            <span class="timebox">
+                {{ entry.start_dt|format_time(timezone=entry.event.tz) }} - {{ entry.end_dt|format_time(timezone=entry.event.tz) }}
+            </span>
+        </p>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_time(entry) %}
+    <div class="time">
+        <div class="time-from">
+            {{ entry.start_dt|format_time(timezone=entry.event.tz) }}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-to">
+            {{ entry.end_dt|format_time(timezone=entry.event.tz) }}
+        </div>
+    </div>
+{% endmacro %}
+
+{# Utilities #}
+
+{% macro render_place(entry) %}
+    {% set entry = entry.object if entry.object else entry %}
+    {% set place_arr = [entry.venue_name or '', entry.room_name or '', entry.address or ''] %}
+
+    {% for el in place_arr %}
+        {% if el %}
+            {{ el + ', ' if loop.nextitem else el }}
+        {% endif %}
+    {% endfor %}
+{% endmacro %}
+
+{% macro render_names(people, show_title=false, show_affiliation=false) %}
+    {% for person in people %}
+        {{
+            person.get_full_name(show_title=show_title, abbrev_first_name=false, last_name_first=false, last_name_upper=false) +
+            (' (' + person.affiliation + ')' if show_affiliation and person.affiliation else '') +
+            (',' if not loop.last else '')
+        }}
```

Comment:
Try this, which should do the same since the only place where whitespace is significant is before the `,`:

```jinja
        {{ person.get_full_name(show_title=show_title, abbrev_first_name=false, last_name_first=false, last_name_upper=false) }}
        {%- if show_affiliation and person.affiliation %}
            ({{ person.affiliation }})
        {%- endif -%}
        {%- if not loop.last %},{% endif %}
```

Reply:
Works!